rmarkdown/research/comparisons-with-existing-tools.md

```diff
@@ -0,0 +1,54 @@
+# Comparing rMarkdown with existing tools
+
+How does rMarkdown compare with existing tools in DataMade's stack or possible alternatives.
+
+## Pweave
+
+Like rMarkdown, [Pweave](http://mpastell.com/pweave/) is an implementation of [noweb](https://en.wikipedia.org/wiki/Noweb), but one that primarily targets Python instead of R.
+
+The main advantage of Pweave is that it is Python. 
+
+While rMarkdown does allow for Python code chunks, there is typically some setup code and that does need to in R. With Pweave, it's all Python.
+
+That is really the only advantage.
+
+Like rMarkdown requires an additional runtime beyond standard Python. rMarkdown requires R and Pweave requires
+[IPython](https://ipython.org/).
+
+Pweave is not actively maintained, and has not been updated
+in three years.
+
+rMarkdown has better editor support than Pweave. For the following editors, rMarkdown is as good and usually better
+than support for Pweave, if there any Pweave support exists.
+
+* [sublime](https://packagecontrol.io/packages/knitr)
+* [emacs](https://ess.r-project.org/)
+* [atom](http://www.goring.org/resources/atom_and_r.html)
+* [vscode](https://marketplace.visualstudio.com/items?itemName=Ikuyadeu.r)
+
+rMarkdown also has its own IDE, [RStudio](https://rstudio.com/)
```

Comment:
It's great to hear that RMarkdown has such wide support. Our [existing data analysis guidelines](https://github.com/datamade/data-analysis-guidelines/blob/master/001-setting-up.md#configure-atom) make a strong recommendation on which editor to use, though, and I've heard @hancush express the belief that RStudio is really good and we should recommend it. What do you think?

Reply:
added this recommendation recommendation doc.